autoupgrade.php

```diff
@@ -48,20 +48,24 @@ public function __construct()
             if (defined('PS_ADMIN_DIR')) {
                 define('_PS_ADMIN_DIR_', PS_ADMIN_DIR);
             } else {
-                $this->_errors[] = $this->trans('This version of PrestaShop cannot be upgraded: the PS_ADMIN_DIR constant is missing.', [], 'Modules.Autoupgrade.Admin');
+                $this->_errors[] = $this->trans('This version of PrestaShop cannot be upgraded: the PS_ADMIN_DIR constant is missing.', []);
             }
         }
 
-        $this->displayName = $this->trans('1-Click Upgrade', [], 'Modules.Autoupgrade.Admin');
-        $this->description = $this->trans('Upgrade to the latest version of PrestaShop in a few clicks, thanks to this automated method.', [], 'Modules.Autoupgrade.Admin');
+        $this->displayName = $this->trans('1-Click Upgrade', []);
```

Comment:
```suggestion
        $this->displayName = $this->trans('1-Click Upgrade');
```

Is there a reason to keep the second parameter as the default?

Reply:
there is no real reason to keep the 2nd parameter which is the default value. I will also remove them